Code Quality/Tools.md

```diff
@@ -168,72 +186,90 @@ Infinum created various rulesets for ESLint and TSLint which you can check out [
 
 If you have no issues with prettier SCSS formatting and you decide to use Prettier, it is recommended to install [stylelint-prettier](https://github.com/prettier/stylelint-prettier) plugin and preset.
 
+```js
+{
+  "plugins": ["stylelint-prettier"],
+  "rules": {
+    "prettier/prettier": true
+  }
+}
+```
+
+If you want to use stylelint, it is recommended to use [@infinumjs/stylelint-config](https://github.com/infinum/stylelint-config). In order to use it with prettier, please add [stylelint-config-prettier](https://github.com/prettier/stylelint-config-prettier).
+
+```js
+{
+  "extends": [
+    "@infinumjs/stylelint-config",
+    "stylelint-config-prettier" // needed if you use prettier to format SCSS
+  ]
+}
+```
+
 ### Putting it all together
 
-Here is the complete example which runs TypeScript compilation check on all files, prettier, tslint and stylelint on an Angular (v10) project:
+Here is the complete example which runs TypeScript compilation check on all files, prettier, eslint and stylelint on an Angular (v10) project:
 
 ```js
-// package.json
 {
   "scripts": {
-    "lint:ng": "ng lint",
-    "tsc": "concurrently \"npm run tsc:app\" \"npm run tsc:spec\"",
```

Comment:
tsc-related scripts should still be here and you should add `npm run tsc` to husky pre-commit hook (in the new format in .husky/pre-commit)